docs/use-cases/observability/clickstack/host-logs/generic-host-logs.md

```diff
@@ -0,0 +1,480 @@
+---
+slug: /use-cases/observability/clickstack/integrations/host-logs
+title: 'Monitoring Host Logs with ClickStack'
+sidebar_label: 'Host Logs'
+pagination_prev: null
+pagination_next: null
+description: 'Monitoring Host Logs with ClickStack'
+doc_type: 'guide'
+keywords: ['host logs', 'systemd', 'syslog', 'OTEL', 'ClickStack', 'system monitoring']
+---
+
+import Image from '@theme/IdealImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import log_view from '@site/static/images/clickstack/host-logs/log-view.png';
+import search_view from '@site/static/images/clickstack/host-logs/search-view.png';
+import import_dashboard from '@site/static/images/clickstack/import-dashboard.png';
+import logs_dashboard from '@site/static/images/clickstack/host-logs/host-logs-dashboard.png';
+import finish_import from '@site/static/images/clickstack/host-logs/import-dashboard.png';
+import { TrackedLink } from '@site/src/components/GalaxyTrackedLink/GalaxyTrackedLink';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Monitoring Host Logs with ClickStack {#host-logs-clickstack}
+
+:::note[TL;DR]
+This guide shows you how to monitor host system logs with ClickStack by configuring the OpenTelemetry collector to ingest syslog files. You'll learn how to:
```

Comment:
we want to do more than syslog, there are other files that are useful - see https://clickhouse.com/docs/use-cases/observability/clickstack/getting-started/local-data#create-otel-configuration

Check we also collect these